src/Raven.Server/Documents/Revisions/RevisionsStorage.cs

```diff
@@ -581,7 +581,7 @@ private static BlittableJsonReaderObject RevertSnapshotFlags(DocumentsOperationC
             return moreRevisionToDelete && deletedRevisionsCount == numberOfRevisionsToDelete;
         }
 
-        public void DeleteRevisionsFor(DocumentsOperationContext context, string id, long? maximumRevisionsToDeleteUponDocumentUpdate = long.MaxValue)
+        public void DeleteRevisionsFor(DocumentsOperationContext context, string id, long? maximumRevisionsToDeleteUponDocumentUpdate = long.MaxValue, bool artificial = false)
```

Comment:
you want to have `DocumentFlags flags = DocumentFlags.None` as an argument

Reply:
done